pulpsolv.py

```diff
@@ -286,11 +299,47 @@ class RpmUnitSolvableFactory(BasetUnitSolvableFactory):
     ]
 
 
-def load_repo_units(plugin_manager, repo_name, factory):
+class ErratumSolvableFactory(BasetUnitSolvableFactory):
+    attribute_factories = [
+        # cargo-culting from
+        # https://github.com/openSUSE/libsolv/blob/master/ext/repo_updateinfoxml.c
+        AttributeFactory('errata_id', target_attr='name',
+                         conversion=lambda x: str('patch:' + x)),
```

Comment:
can we actually name them like `errata:` or something like this? Because "patch" means to patch your RPMs and not to install them. Also your lambda can look like `lambda x: 'errata:{}'.format(x)`

Reply:
sure thing...